packages/config/config/devices/0x010f/fgd212_3.5.json

```diff
@@ -98,15 +100,15 @@
 		},
 		"10": {
 			"label": "Timer functionality (auto - off)",
-			"description": "Defines the setting of the timer",
+			"description": "Will automatically turn off after the specified time from switching on (0 = disable)",
```

Comment:
The special value `0` should instead be an `"option"`. With `allowManualEntry` set to `true` (the default), this is interpreted as a suggestion in addition to the free entry.

Reply:
Ok, I was unsure if this was to be used even if the value 0 had a totally different meaning from the others.